autogpt/setup.py

```diff
@@ -192,8 +177,10 @@ def generate_aiconfig_automatic(user_prompt) -> AIConfig:
             "content": system_prompt,
         },
         {
-            "role": "user",
-            "content": f"Task: '{user_prompt}'\nRespond only with the output in the exact format specified in the system prompt, with no explanation or conversation.\n",
+            "role":
+            "user",
+            "content":
+            Template(TASK_PROMPT_AICONFIG_AUTOMATIC).render(user_prompt=user_prompt)
```

Comment:
create a variable prompt_ai_config_automatic just so it's more clear.
Then call 

```
  messages = [
Expand All
	@@ -192,8 +177,10 @@ def generate_aiconfig_automatic(user_prompt) -> AIConfig:
            "content": system_prompt,
        },
        {
            "role":
            "user",
            "content": prompt_ai_config_automatic
    },
    ]

```